fdbserver/IPager.h

```diff
@@ -40,6 +40,13 @@ typedef uint32_t PhysicalPageID;
 typedef uint32_t QueueID;
 #define invalidQueueID std::numeric_limits<QueueID>::max()
 
+// Pager Events
+enum class events{ pagerCacheLookup = 0, pagerCacheHit, pagerCacheMiss, pagerWrite, MAXEVENTS};
```

Comment:
General comment for the PR, to match the style of the rest of the FDB code base, data types should begin with a capital letter - `Events` instead of `events` etc.  

Also, let's match these two type names, for example 'PagerEvents` and `PagerEventReasons`.